index.js

```diff
@@ -20,9 +20,10 @@ class BirthdayReminder extends q.DesktopApp {
 
         if (currentMonth == monthOfTheBirthday && currentDay == dayOfTheBirthday) {
             logger.info("BirthdayReminder, you will never forget a birthday.");
+            const color = '#00FF00';
```

Comment:
If you don't use the color from the config, remove it as well from the questions in the package.json